custom_title_field.php

```diff
@@ -0,0 +1,83 @@
+<?php
```

Comment:
rename to title.php

Reply:
I already have a file named title.php, should I rename it to something else?

Reply:
That's okay, just use the template-parts and custom-field folders.